### PR TITLE
Added method to check if a class exists

### LINKED ISSUE #232: Add method to check if a class exists
Add a function that returns if a given class exists in the schema

Proposal signature:
`client.schema.exists(className: str) ->bool`

### PR DESCRIPTION
**Pull request by meetagrawal09**
This PR fixes #232.

### CODE REVIEW

**Comment by meetagrawal09** [comment]
> To avoid any confusion in the future about your contribution to Weaviate, we work with a [Contributor License Agreement](https://weaviate.io/service/contributor-license-agreement). If you agree, you can simply add a comment to this PR that you agree with the CLA so that we can merge.beep boop - the Weaviate bot 👋🤖PS:Are you already a member of the [Weaviate Slack channel?](https://weaviate.io/slack)

I agree with the Contributor License Agreement.

File: integration/test_schema.py [hunk 1]
```diff
@@ -36,8 +36,13 @@ def test_create_class_with_implicit_and_explicit_replication_factor(
         }
 
     client.schema.create_class(single_class)
+    class_exists = client.schema.exists("Barbecue")
+    class_does_not_exist = client.schema.exists("Not-Barbecue")
     created_class = client.schema.get("Barbecue")
+    
     assert created_class["class"] == "Barbecue"
+    assert class_exists
+    assert not class_does_not_exist
```
Comment (dirkkul) [inline integration/test_schema.py:45]: Would you create an extra test for this?

I'd rather not mix different things. Also please assert directly eg
```
assert client.schema.exists("Barbecue")
assert not client.schema.exists("Not-Barbecue")
```

File: weaviate/schema/crud_schema.py [hunk 2]
```diff
@@ -352,6 +352,29 @@ def update_config(self, class_name: str, config: dict) -> None:
...
+            True if the class exists, False otherwise.
+        
+        Examples
+        --------
+        >>> client.schema.exists('Author')
+        True
+        """
+        
+        schema = self.get(class_name=class_name)
+        return len(schema["classes"]) > 0
```
Comment (dirkkul) [inline weaviate/schema/crud_schema.py:376]: this doesn't work as you already got the specific class, eg the return is not the complete schema

**Review by dirkkul — CHANGES_REQUESTED** [review]

**Comment by StefanBogdan** [comment]
Since this was a priority we added this feature already, closing.

**Comment by hsm207** [comment]
@StefanBogdan which PR implemented this feature?

**Comment by StefanBogdan** [comment]
@hsm207 this one: https://github.com/weaviate/weaviate-python-client/pull/311

**Pull request closed without merging**